eng/pipelines/common/insertion.yml

```diff
@@ -0,0 +1,35 @@
+stages:
+  - template: vs-insertion/stage/v1.yml@xamarin-templates
```

Comment:
@pjcollins  @mandel-macaque  do we need the 2 insertions? the old and new one?

Reply:
I'd recommend keeping it around for now, unless there is a burden in doing so? It may be useful to have in some cases still.

Reply:
ok thanks! we will keep it